USERS.md

```diff
@@ -28,3 +29,5 @@ Organizations below are **officially** using Argo Rollouts. Please send a PR wit
 1. [Twilio SendGrid](https://sendgrid.com)
 1. [Ubie](https://ubie.life/)
 1. [VISITS Technologies](https://visits.world/en)
+1. [127Labs](https://127labs.com)
```

Comment:
I'd prefer creating a separate PR for this. It doesn't make any sense for this change to go in with this PR for adding TCP routes.

Reply:
I'm just making sure the items in the checklist are addressed, I will remove this. I suggest updating the checklist.